docs/customization/template.rst

```diff
@@ -155,9 +168,27 @@ on the create action of Resources, at the bottom of the page (after the content
 
 .. tip::
 
-    In order to find events in Sylius templates you can simply search for the ``sonata_block_render_event`` phrase in your
+    In order to find events in Sylius templates you can simply search for the ``sylius_template_event`` phrase in your
     project's directory.
 
+How to locate event in browser?
```

Comment:
In this place, we should also mention the profiler section that shows the rendered events, blocks and their templates with priorities:

![Zrzut ekranu 2022-07-4 o 08 33 30](https://user-images.githubusercontent.com/6212718/177095404-77e5ee5d-acf1-4761-9a6f-4638cf603a1f.png)